Package.swift

```diff
@@ -51,6 +51,7 @@ let package = Package(
       name: "TestingTests",
       dependencies: [
         "Testing",
+        "_Testing_CoreGraphics",
```

Comment:
It can be done via a separate PR, but just a reminder to add this new target/module to CMake too, following the recent pattern for the Foundation cross-import overlay.

Reply:
Will address separately.